Frontend/src/Components/Notifications/Notifications.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import { useSocket } from "../Contexts/SocketContext";
+import Loading from "../Loading/Loading.jsx";
+import "./Notifications.css";
+
+const API_KEY = import.meta.env.VITE_BACKEND_ADDRESS;
+
+const Notifications = () => {
+  const { socket } = useSocket();
+  const [notifications, setNotifications] = useState([]);
+  const [isLoading, setLoading] = useState(true);
```

Comment:
did you want to use the loading hook you made here?

Reply:
oh yess!